spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -17,51 +17,35 @@
 package org.springframework.cloud.gcp.storage;
 
 import com.google.cloud.storage.Storage;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 
-import org.springframework.beans.factory.InitializingBean;
-import org.springframework.core.io.DefaultResourceLoader;
 import org.springframework.core.io.ProtocolResolver;
 import org.springframework.core.io.Resource;
 import org.springframework.core.io.ResourceLoader;
 import org.springframework.util.Assert;
 
 /**
+ * A {@link ProtocolResolver} implementation for the {@code gs://} protocol.
+ * 
```

Comment:
That's not a Checkstyle issue. The real problem that Eclipse formatter doesn't have ability to remove trailing space. Native Eclipse requires special plugin on the matter: http://andrei.gmxhome.de/anyedit/

Not sure what we can do to unless manually switch to the IDEA formatter after having Checkstyle report.

OTOH if we all use IDEA, there might not be reason to use this Eclipse formatter but configure the IDEA native to closer style.

(I'll fix all whitespaces issue in this PR eventually)